examples/directml/stable_diffusion_xl/config_vae_decoder.json

```diff
@@ -79,7 +79,43 @@
                         "GroupNorm": [0, 1, 2]
                     }
                 },
-                "force_fp32_ops": ["RandomNormalLike"]
+                "force_fp32_ops": ["RandomNormalLike"],
+                "force_fp32_nodes": [
+                    "/decoder/up_blocks.2/upsamplers.0/conv/Conv",
+                    "/decoder/up_blocks.3/resnets.0/conv_shortcut/Conv",
+                    "Transpose_NCHW_to_NHWC_22",
```

Comment:
Hmm you're probably not using the most recent branch from main or ORT 1.16. These transposes should not be there anymore because `group_norm_channels_last` is set to false.